types/types.go

```diff
@@ -101,6 +103,7 @@ const (
 	LonghornLabelRecurringJobGroup        = "job-group"
 	LonghornLabelOrphan                   = "orphan"
 	LonghornLabelOrphanType               = "orphan-type"
+	LonghornLabelRecoveryBackendConfigMap = "recovery-backend"
```

Comment:
Suggest renaming to `LonghornLabelRecoveryBackend` because configmap is implementation detail, probably no need to disclose in this const.

Reply:
Updated. Thank you.